mne/chpi.py

```diff
@@ -837,8 +835,19 @@ def compute_chpi_amplitudes(raw, t_step_min=0.01, t_window='auto',
        sinusoidal amplitudes to MEG channels.
        It uses SVD to determine the phase/amplitude of the sinusoids.
 
-    The output is meant to be used with :func:`~mne.chpi.compute_chpi_locs`.
+    The number of fitted points ``n_pos`` will depend on the velocity of head
+    movements as well as ``t_step_max`` (and ``t_step_min`` from
+    :func:`mne.chpi.compute_chpi_amplitudes`).
```

Comment:
It looks like this paragraph should not have been moved (though the one below indeed should have)

Reply:
If I recall, there seems to be no explicit `n_pos` in the code so this through me off.  I see there is a reference to head velocity so your comment makes sense.